### DISCUSSION TITLE
Audi A4 B9 2017 M.I.B. carplay, map update

### DISCUSSION THREAD

**Discussion by Gintazas** [Mr-MIBonk/M.I.B._More-Incredible-Bash#230] [Q&A]
Hello. In my audi was version MHS2_ER_AU_P0470 MU Software:0191 GEM 4.5s 
Did i right update to version MHS2_ER_AU_P1242 MU 0213 becouse GEM got down version 4.12s? Why it happend? In to what correct version i should have to  update my radio to have M.I.B? want to have CARPLAY function ant to update maps. When i try to install MIB, "START UPDATE"is not active
Thanks for help and solutions
![IMG_20230524_113338](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/48887740/cfe226dc-19a4-430b-9ade-14c1d741eb7d)
![IMG_20230525_193942](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/48887740/c4c95b36-656b-49a2-8314-510f11f6ec47)
![IMG_20230525_194456](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/48887740/d9514560-46d1-4a63-bdf3-803889053751)
![IMG_20230526_123543](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/48887740/e0afefb2-ebf7-4bf5-bc0e-89c6497f81cd)

**Comment by harman-f** [comment]
Wrong FW
We only deal in MHI2 here
